CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+# v4.0.0
```

Comment:
n_: Can we change this to "Unreleased" for now? Possibly update it to 4.0.0, after we test out the 4.0.0-rc2 build is working as expected.

Reply:
done.